lib/nexthop_group.c

```diff
@@ -387,13 +401,6 @@ DEFPY(ecmp_nexthops, ecmp_nexthops_cmd,
 	struct nexthop *nh;
 	bool legal;
 
-	/*
```

Comment:
How is this not a problem still?

Reply:
I removed the `assert` because `addr` might be NULL if a nexthop address is not given (this should be possible now).